src/raster/r.connectivity/r.connectivity.distance/r.connectivity.distance.py

```diff
@@ -406,9 +406,8 @@ def main():
     # cost_ewres = reg['ewres']
 
     # Rasterize patches
-    # http://www.gdal.org/gdal_tutorial.html
-    # http://geoinformaticstutorial.blogspot.no/2012/11/convert-
-    # shapefile-to-raster-with-gdal.html
+    # https://web.archive.org/web/20180707051311/http://gdal.org/gdal_tutorial.html
```

Comment:
```suggestion
    # https://gdal.org/en/latest/tutorials/index.html
```

Reply:
Which of the tutorials specifically? I went back to that date (content similar to when committed) as it was more some content on rasters, and wasn't split up yet. Can you infer what tutorial is meant to show, what info is linked?

Reply:
Ok, we could use that, but I don't see, even historically what pattern it referred to (maybe I didn't go far enough).

Reply:
```suggestion
    # https://gdal.org/en/stable/tutorials/raster_api_tut.html
```